docs/migrations/web-react/MIGRATION-v2.md

```diff
@@ -98,6 +98,23 @@ Please, add the `danger` icon to your project's assets.
 
 Either install newer version of the `spirit-icons` package or add `danger` named icon to your project's icons.
 
+### Alert: `role="alert"`
+
+The `role="alert"` has been removed for the default setting of the component.
+
+The alert role is used to communicate an important and usually time-sensitive message to the user. When this role is added to an element,
+the browser will send out an accessible alert event to assistive technology products which can then notify the user.
+The alert role should only be used for information that requires the user's immediate attention, which is typically
+content that is dynamically displayed, not for content that appears on page load. It is perfect for situations such
+as when a user fills out a form and JavaScript is used to add an error message - the alert would immediately read out
```

Comment:
`alert would immediately read out` sounds a little bit odd to me. The "alert" would "read out" the  message? Did you mean that some kind of assistive technology will read the message. I am a little bit confused what you are trying to say. 🤔

Reply:
I used the part of the text that was in the issue description, but I agree that it is not entirely obvious. I will edit it.

Reply:
Better, thanks. 👍